tools/build-from-source.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -o xtrace
+apt-get update
+apt-get install make
+cd /tmp
+ls -lh
+lsb_release -a
+cd /tmp/ubuntu-advantage-client
+make deps
+dpkg-buildpackage -us -uc
+ls /tmp |grep ubuntu-advantage-tools
```

Comment:
I don't think we need this file as it doesn't really have purpose for typical development, we can create in on the fly in behave.

Reply:
ok , deleted it